tox.ini

```diff
@@ -20,7 +20,7 @@ envlist =
 commands =
     {env:_PYTEST_TOX_COVERAGE_RUN:} pytest {posargs:{env:_PYTEST_TOX_DEFAULT_POSARGS:}}
     coverage: coverage combine
-    coverage: coverage report
+    coverage: coverage report -m
```

Comment:
or we could add `show_missing = true` into `.coveragerc` 🤔

Reply:
Yes, sounds good - although it cannot be disabled/skipped then AFAIK (i.e. if you are using `coverage report` yourself).